assets/js/googlesitekit/widgets/register-defaults.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Widgets API defaults
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { CONTEXT_DASHBOARD, CONTEXT_PAGE_DASHBOARD } from './default-contexts';
+import {
+	AREA_ALL_TRAFFIC,
+	AREA_SEARCH_FUNNEL,
+	AREA_POPULARITY,
+	AREA_SPEED,
+	AREA_EARNINGS,
+} from './default-areas';
+
+/**
+ * Defines default widget areas for a given context.
+ *
+ * @since n.e.x.t
+ *
+ * @param {Object} widgetsApi Widgets API.
+ */
+export function registerDefaults( widgetsApi ) {
+	const contexts = [ CONTEXT_DASHBOARD, CONTEXT_PAGE_DASHBOARD ];
```

Comment:
This approach of relying on the `widgetsApi` object like 3P would looks good to me 👍 

We should probably name the variable `Widgets` though, which is the name we use elsewhere for these objects that will in production end up as e.g. `global.googlesitekit.widgets`.

Reply:
Updated.